LocationSimulator/ViewController/MapViewController/MapViewController.swift

```diff
@@ -260,20 +256,24 @@ class MapViewController: NSViewController {
     @discardableResult
     func connectDevice() -> Bool {
         // Make sure we have a device to connect, which is not already connected. We need a window to show errors.
-        guard !self.deviceIsConnectd, let device = self.device, let window = self.view.window else { return false }
+        guard !self.deviceIsConnected, let devices = self.devices, let window = self.view.window else { return false }
         do {
             // Show the error indicator and a progress spinner.
             self.contentView?.showErrorInidcator()
             // If the pairing and uploading of the developer disk image is successfull create a spoofer instance.
-            try device.pair()
+            if self.devices != nil {
+                for device in self.devices! {
+                    try device.pair()
+                }
+            }
             // Hide the error indicator if the device was connected sucessfully.
             self.contentView?.hideErrorInidcator()
             // We successfully connected the device.
-            self.deviceIsConnectd = true
+            self.deviceIsConnected = true
         } catch let error {
             // Stop the spinner even if an error occured.
             self.contentView?.stopSpinner()
-            self.deviceIsConnectd = false
+            self.deviceIsConnected = false
```

Comment:
Not sure if we should try to disconnect any successfully connected devices here?